caddyhttp/httpserver/plugin.go

```diff
@@ -447,6 +447,8 @@ var directives = []string{
 	"realip", // github.com/captncraig/caddy-realip
 	"git",    // github.com/abiosoft/caddy-git
 
+	"cache",
```

Comment:
No, don't reorder any of the other directives. Just leave cache where it best belongs, which after your investigation, is apparently in this place. Just remove the whitespace so it fits in the group. Thanks!